tests/lib/config/config-file.js

```diff
@@ -155,6 +156,37 @@ describe("ConfigFile", function() {
 
         });
 
+        it("should throw an error when extends config is not found", function() {
```

Comment:
I spent a couple hours trying to figure out how to better test these changes and couldn't come up with anything. I agree more tests are better, I just don't know how to do so. Suggestions welcome.